README.md

```diff
@@ -64,6 +64,10 @@ patches `require` + `autoload` to use these absolute paths, thereby avoiding hav
 
 Problem outlined in this [talk](https://www.youtube.com/watch?v=kwkbrOwLsZY)
 
+## Troubleshooting
+
+If you're experiencing problems with loading your application and you're unable to successfully run `Bootscale.regenerate`, delete the `tmp/bootscale` folder.
```

Comment:
Yeah, it's gone. I can try replicating it on the weekend, but I promise nothing.

I've updated the PR